profiling/src/exporter/mod.rs

```diff
@@ -106,15 +112,24 @@ impl Request {
 }
 
 impl ProfileExporter {
-    pub fn new<IntoCow: Into<Cow<'static, str>>>(
-        family: IntoCow,
+    pub fn new<F, N, V>(
+        profiling_library_name: N,
+        profiling_library_version: V,
+        family: F,
         tags: Option<Vec<Tag>>,
         endpoint: Endpoint,
-    ) -> anyhow::Result<ProfileExporter> {
+    ) -> anyhow::Result<ProfileExporter>
+    where
+        F: Into<Cow<'static, str>>,
```

Comment:
why do we need to alias to different types ? 

Also the `Cow 'static` lifetime seems strange to me.